scripts/create-triggers-accounts.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env sh
+
+set -ex
+
+################################################
+#                                              #    
+# WARNING: Deleting your local fxa database!!! #
+#                                              #    
+################################################
+
+# Initialize a new, empty database.
+mysql fxa -e 'DROP DATABASE IF EXISTS fxa'
+node ./bin/db_patcher.js
```

Comment:
Technically, it doesn't need to do this delete. It could just start from whatever is already there and work.